documentation/src/docs/dev-guide.md

```diff
@@ -0,0 +1,203 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Flink Connector - Dev Guide
+
+Learn how to build your own applications that using Flink connector for Pravega.
+
+ 
+# Prerequisites
+To complete this guide, you need:
```

Comment:
newline required between this line and the first bullet in order for bullet points to render propertly in MkDocs